app/controllers/actions/ValidateMessageAction.scala

```diff
@@ -34,8 +34,7 @@ class ValidateMessageAction @Inject()(xmlValidationService: XmlValidationService
       case body: NodeSeq =>
         if (body.nonEmpty) {
           val rootElementName = body.head.label
-
-          XSDFile.Arrival.SupportedMessages.get(rootElementName) match {
+          XSDFile.SupportedMessages.get(rootElementName) match {
```

Comment:
This is incorrect as this will allow users to submit a request of release to the arrivals backend and unloading remarks to the departures backend.<br><br>There should be a ValidateArrivalMessageAction and ValidateDepartureMessageAction and the call should be XSDFile.Arrival.SupportedMessages.get(rootElementName) for arrivals and XSDFile.Departure.SupportedMessages.get(rootElementName) for departures.

Reply:
There also needs to be tests for ValidateArrivalMessageAction and ValidateDepartureMessageAction.